conanfile.py

```diff
@@ -264,7 +263,7 @@ def build_cmake(self):
         elif self.settings.os == "Windows":
             cmake.definitions["DIRECTX"] = self.options.directx
 
-        cmake.configure(build_dir='build')
+        cmake.configure(build_dir=self.install_folder)
```

Comment:
self.install_folder doesn't seem to be defined (linter gave me the hint on export)

Reply:
Idk, but while debugging the conan SDL2 build process, self.source_folder, self.build_folder, self.install_folder and self.package_folder are available for SDL2Conan.
Those seem to be the only member variables of SDL2Conan with file paths.

Reply:
https://docs.conan.io/en/latest/reference/conanfile/attributes.html#folders